wordpress-mobile/WordPress-iOS
# Login: fix Google button covering error label

Fixes #7898 

![simulator screen shot - iphone 8 - 2017-10-03 at 11 07 51](https://user-images.githubusercontent.com/517257/31138268-224ba9c2-a82b-11e7-95dd-ab529078e331.png)

To test:
- ensure the input field and labels still work as before
- ensure the error label can't be covered by the Google button

Needs review: @aerych 
*Note:* forked from #7887
